docs/autogen_shortcuts.py

```diff
@@ -5,6 +5,10 @@
 
```

Comment:
I haven't had a chance to look into this script yet, so I won't try to review changes to it - at some point I'll look through it as a whole.